doc/classes/EditorNode3DGizmo.xml

```diff
@@ -9,6 +9,13 @@
 	<tutorials>
 	</tutorials>
 	<methods>
+		<method name="_begin_handle_action" qualifiers="virtual">
+			<return type="void" />
+			<param index="0" name="id" type="int" />
+			<param index="1" name="secondary" type="bool" />
+			<description>
```

Comment:
Would appreciate any details on what this should do, as I don't know what it does, the base method isn't exposed so no information is available

Reply:
CC @KoBeWi who added this AFAICT.

Reply:
It was added in #71092
It's called when a user clicks the gizmo and can be used to fetch and store initial values.